test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13182,26 +13184,15 @@ def run(self):
                 return None
             self.logger.debug(_p.after.decode())
             for _l in _sc:
-                self.logger.debug('sending: ' + _l[0])
-                _p.sendline(_l[0])
-                time.sleep(_st)
-                # only way I could figure out to catch a sleep command
-                # within a spawned pexpect child. Might need revisiting
-                if 'sleep' in _l[0]:
-                    _secs = _l[0].split()[1]
-                    self.logger.debug('sleeping ' + str(_secs))
-                    time.sleep(float(_secs))
-                if len(_l) > 1:
-                    for _r in range(1, len(_l)):
-                        self.logger.debug('expecting: ' + _l[_r])
-                        _p.expect(_l[_r])
-                        time.sleep(_st)
-                        self.logger.debug('received: ' + _p.after.decode())
-                    time.sleep(_st)
-                    self.logger.debug('received: ' + _p.after.decode())
-            self.logger.debug('sending Ctrl-D')
-            _p.sendcontrol('d')
-            time.sleep(_st)
+                (cmd, out) = _l
+                self.logger.debug('sending: ' + cmd)
```

Comment:
Hey Kumar, thanks for walking me through the changes! 
As discussed, please print these out at a different log level so that pbs_benchpress output prints these out by default. The rest looks fine to me. Thanks again!